README.md

```diff
@@ -33,16 +33,16 @@ definitions are axiom-free.
 - [Installation](#installation)
 - [HoTT Variant](#hott-variant)
 
-## Live demo
-
-Try it now in your browser with [JSCoq](http://mattam82.github.io/Coq-Equations/assets/jsexamples/equations_intro.html)!
-
```

Comment:
I think it's good to keep the live demo separately. People will not expect to find it under "Documentation". You can still leave the link you added below too, doesn't hurt.

Reply:
I have just moved it below with the refman so that people don't see the old version first

Reply:
I only see the `Live demo` section in red, not in green in the diff, so to me it seems gone. I think anyway that it should come first.

Reply:
The old live demo is now linked L45. 
The new interactive tutorials are now the first point on the list. 
Do you prefer it to be put forward in a first independent section as before ?

Reply:
Yes, I think it would be best indeed. Tutorials are what people will likely look for first, so it seems a good idea to put them in front and highlight them. :)